api/specs/kmd.oas3.json

```diff
@@ -0,0 +1,1907 @@
+        "x-go-package": "github.com/algorand/go-algorand/daemon/kmd/lib/kmdapi"
+      },
+      "APIV1POSTMultisigProgramSignResponse": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "boolean",
+            "x-go-name": "Error"
+          },
+          "message": {
+            "type": "string",
+            "x-go-name": "Message"
+          },
+          "multisig": {
+            "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
+            "type": "string",
+            "format": "byte",
+            "x-go-name": "Multisig"
+          }
+        },
+        "description": "APIV1POSTMultisigProgramSignResponse is the response to `POST /v1/multisig/signdata`\nfriendly:SignProgramMultisigResponse",
+        "x-go-package": "github.com/algorand/go-algorand/daemon/kmd/lib/kmdapi"
+      },
+      "APIV1POSTMultisigTransactionSignResponse": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "boolean",
+            "x-go-name": "Error"
+          },
+          "message": {
+            "type": "string",
+            "x-go-name": "Message"
+          },
+          "multisig": {
+            "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
+            "type": "string",
+            "format": "byte",
+            "x-go-name": "Multisig"
+          }
+        },
+        "description": "APIV1POSTMultisigTransactionSignResponse is the response to `POST /v1/multisig/sign`\nfriendly:SignMultisigResponse",
+        "x-go-package": "github.com/algorand/go-algorand/daemon/kmd/lib/kmdapi"
+      },
+      "APIV1POSTProgramSignResponse": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "boolean",
+            "x-go-name": "Error"
+          },
+          "message": {
+            "type": "string",
+            "x-go-name": "Message"
+          },
+          "sig": {
+            "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
+            "type": "string",
+            "format": "byte",
+            "x-go-name": "Signature"
+          }
+        },
+        "description": "APIV1POSTProgramSignResponse is the response to `POST /v1/data/sign`\nfriendly:SignProgramResponse",
+        "x-go-package": "github.com/algorand/go-algorand/daemon/kmd/lib/kmdapi"
+      },
+      "APIV1POSTTransactionSignResponse": {
+        "type": "object",
+        "properties": {
+          "error": {
+            "type": "boolean",
+            "x-go-name": "Error"
+          },
+          "message": {
+            "type": "string",
+            "x-go-name": "Message"
+          },
+          "signed_transaction": {
+            "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
+            "type": "string",
+            "format": "byte",
+            "x-go-name": "SignedTransaction"
```

Comment:
@joe-p  do you know if this is the same signed txn object used in algod? If yes i can align it by adding out own x algokit vendor extension here so generators embed the model from transact crate/package. Otherwise we can keep as is